osu.Game.Rulesets.Osu/Objects/SpinnerBonusTick.cs

```diff
@@ -18,7 +18,7 @@ public SpinnerBonusTick()
 
         public class OsuSpinnerBonusTickJudgement : OsuSpinnerTickJudgement
         {
-            protected override int NumericResultFor(HitResult result) => 1100;
+            protected override int NumericResultFor(HitResult result) => 50;
```

Comment:
Maybe extract a few constants? The name would be like e.g.
```suggestion
            protected override int NumericResultFor(HitResult result) => SPINNER_BONUS_TICK_VALUE + SPINNER_TICK_VALUE;
```

Reply:
considered this but not sure it's necessary. at that point we should just construct a judgement in the test